cf-deployment.yml

```diff
@@ -717,8 +717,9 @@ instance_groups:
             authorized-grant-types: client_credentials
             secret: "((uaa_clients_cf_smoke_tests_secret))"
           cloud_controller_username_lookup:
-            authorities: scim.userids
+            authorities: scim.userids,scim.read
             authorized-grant-types: client_credentials
+            override: true
```

Comment:
Why is override required? (I'm not very familiar with this UAA logic)

Reply:
We're proposing `override: true` parameter to update `cloud_controller_username_lookup` value in the existing, already deployed manifests. It is our understanding that the default behavior will copy this value to the new deployments only.